definitions/npm/ramda_v0.26.x/flow_v0.104.x-/test_ramda_v0.26.x_list.js

```diff
@@ -825,24 +825,36 @@ const str: string = "hello world";
   }))([1, 2, 3])(["1", "2", "3"]);
 
   describe('uniq', () => {
+    const readOnlyNumbers: $ReadOnlyArray<number> = [1,1,2,3,4,3];
```

Comment:
Lets keep our tests self contained if we can. Can we move this into the relevant test(s)?

Reply:
Sure thing.

Reply:
Done